pkg/apis/pipeline/v1/pipeline_validation_test.go

```diff
@@ -87,7 +87,36 @@ func TestPipeline_Validate_Success(t *testing.T) {
 				Tasks: []PipelineTask{{Name: "foo", TaskRef: &TaskRef{Name: "bar", Kind: ClusterTaskRefKind}}},
 			},
 		},
-	}}
+	}, {
+		name: "valid task with pipelineRef",
```

Comment:
I think we add a negative test to `TestPipeline_Validate_Failure` also for specifying both the `pipelineRef` and `pipelineSpec` at the same time. This is for `expected exactly one, got multiple`.

Reply:
Thanks @khrm Its been added at https://github.com/tektoncd/pipeline/pull/7055/files#diff-82c2295aa9f6136ca65b82043eeb2928c7b81f9374d246d9b202f50a099a187fR224